modules/nf-core/mageck/test/tests/main.nf.test

```diff
@@ -0,0 +1,91 @@
+nextflow_process {
+
+    name "Test Process MAGECK_TEST"
+    script "../main.nf"
+    process "MAGECK_TEST"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mageck"
+    tag "mageck/test"
+
+    test("mageck-test - csv") {
+
+        config "./nextflow.config"
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:false ], // meta map
+                        file(params.modules_testdata_base_path + 'genomics/mus_musculus/mageck/count_table.csv', checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                            process.out.versions,
+                            file(process.out.gene_summary.get(0).get(1)).readLines()[0],
+                            file(process.out.sgrna_summary.get(0).get(1)).readLines()[0],
+                            file(process.out.r_script.get(0).get(1)).readLines()[0]
+                            ).match()
+                }
+            )
+        }
+
+    }
+
+    test("mageck-test-day0 - csv") {
```

Comment:
@SPPearce this test (day0) seems to not produce gene_summary or sgrna_summary output files